docs/content/direct/core-chart.md

```diff
@@ -86,7 +86,7 @@ InstallPCHs: true
 
 # List the Inventory and Transport Spaces (ITSes) to be created by the chart
 # Each ITS consists of a mandatory unique name and an optional type, which could be either host or vcluster (default to vcluster, if not specified)
-ITSes: # ==> installs ocm + ocm-status-addon
+ITSes: # ==> installs ocm (optional) + ocm-status-addon

```

Comment:
I think that your earlier suggestion of how to frame this is better: it is a question of whether OCM is already installed. Having OCM is not optional; the question is whether it is already there. In the case where the user says that OCM is already installed, the chart should check and somehow let the user know if OCM is not actually already there.

Reply:
Or perhaps it is unnecessary to have an ordering requirement that OCM is installed in the ITS _before_ the core Helm chart is instantiated.

Reply:
AFAK, helm cannot verify the existence or wait for a specified condition in the cluster... being the presence of a CRD or the existence/reediness of a pod. As the user needs to go out of his/her way to not install OCM with the chart we can hope that he/she chose with some consideration. We can also add more text to clarify your point in words.